dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/passwordreset/PasswordResetData.java

```diff
@@ -67,6 +70,9 @@
     @Enumerated(EnumType.STRING)
     private Reason reason;
 
+    @EJB
+    SettingsWrapper settingsWrapper;
```

Comment:
We don't want any beans in Entity it is mixing of concerns.  And usually access should be private since we don't wanna use it outside of the class.